packages/vizbuilder/src/components/Sidebar/MeasureSelect.jsx

```diff
@@ -69,9 +69,9 @@ MeasureSelect.defaultProps = {
       props.onClick = handleClick;
       child2 =
         <span className="select-label lead">
-          {item.annotations._cb_sourceName}
+          {item.annotations._cb_sourceName} ({item.annotations._cb_name.replace("_", " ")})
```

Comment:
Not sure what `_cb_name` maps to, but please use the `dataset_name` annotation, with no string replacements, and put it before the `source_name` annotation separated by a hyphen (as requested in the github issue)